stdlib/multiprocessing/connection.pyi

```diff
@@ -1,17 +1,20 @@
 import socket
 import sys
-import types
-from _typeshed import ReadableBuffer
+from _typeshed import Incomplete, ReadableBuffer
 from collections.abc import Iterable
-from typing import Any
+from types import TracebackType
+from typing import Any, Generic, TypeVar
 from typing_extensions import Self, SupportsIndex, TypeAlias
 
 __all__ = ["Client", "Listener", "Pipe", "wait"]
 
 # https://docs.python.org/3/library/multiprocessing.html#address-formats
 _Address: TypeAlias = str | tuple[str, int]
 
-class _ConnectionBase:
+_T1 = TypeVar("_T1")
+_T2 = TypeVar("_T2")
```

Comment:
Calling these `_S` and `_R` or even `_SendT` and `_ReceiveT` might be better for documentation purposes.